### ISSUE TITLE
fastlane hockey action fails on travis

### ISSUE THREAD

**Issue by a1cooke** [fastlane/fastlane#5393]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description
Running fastlane's hockey action on travis is failing with the below error.

##### Complete output when running fastlane, including the stack trace and command used

```
/Users/travis/.rvm/rubies/ruby-2.0.0-p648/lib/ruby/site_ruby/2.0.0/rubygems/core_ext/kernel_require.rb:114:in `require': unable to find a version of 'krausefx-shenzhen' to activate (Gem::LoadError)
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/actions/hockey.rb:16:in `run'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/runner.rb:197:in `block (2 levels) in execute_action'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/actions/actions_helper.rb:35:in `execute_action'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/runner.rb:182:in `block in execute_action'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/runner.rb:181:in `chdir'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/runner.rb:181:in `execute_action'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/runner.rb:107:in `trigger_action_by_name'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/fast_file.rb:140:in `method_missing'
	from Fastfile:73:in `block (2 levels) in parsing_binding'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/lane.rb:33:in `call'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/lane.rb:33:in `call'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/runner.rb:50:in `block in execute'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/runner.rb:46:in `chdir'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/runner.rb:46:in `execute'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/lane_manager.rb:46:in `cruise_lane'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/command_line_handler.rb:30:in `handle'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/commands_generator.rb:51:in `block (2 levels) in run'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/commander-4.3.5/lib/commander/command.rb:153:in `run'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/commander-4.3.5/lib/commander/runner.rb:428:in `run_active_command'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane_core-0.48.1/lib/fastlane_core/ui/fastlane_runner.rb:26:in `run!'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/commander-4.3.5/lib/commander/delegates.rb:15:in `run!'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/commands_generator.rb:244:in `run'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/commands_generator.rb:20:in `start'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/lib/fastlane/cli_tools_distributor.rb:47:in `take_off'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/gems/fastlane-1.97.2/bin/fastlane:5:in `<top (required)>'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/bin/fastlane:23:in `load'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/bin/fastlane:23:in `<main>'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/bin/ruby_executable_hooks:15:in `eval'
	from /Users/travis/.rvm/gems/ruby-2.0.0-p648/bin/ruby_executable_hooks:15:in `<main>'
```

### Configuration Files

Please copy the complete content of your `Fastfile` and any other configuration files you use below:

Fastfile:
```ruby
opt_out_usage

# If you want to automatically update fastlane if a new version is available:
update_fastlane

# This is the minimum version number required.
# Update this, if you use features of a newer version
fastlane_version "1.51.0"

default_platform :ios

platform :ios do

  before_all do
    
    # cocoapods
    # xctool # run the tests of your app

  end

  desc "Responsible for building and signing the app"
  private_lane :build_app do |options|
    sigh

    gym(scheme: "App",
        use_legacy_build_api: true,
        export_method: 'enterprise')

  end

  desc "Test"
  lane :testApp do |options|
      scan(scheme: "App")
  end

  desc "Test the UI"
  lane :testUI do |options|
      scan(scheme: "UiTests",
           devices: ["iPhone 6s"],
           clean: true,
           output_types: "junit")
  end

  desc "Build App and verify compilation"
  lane :verifyBuild do
    build_app
  end

  desc "Build In-House build and upload to Hockey App"
  lane :inhouseRelease do    
    
    build_app
    
    hockey(api_token: ENV['HOCKEY_API_TOKEN'], notify: '0', notes: ENV['RELEASE_NOTES'], notes_type: '0')

  end

  desc "Build Beta build and upload to Hockey App"
  lane :betaRelease do 

    build_app
    hockey(api_token: ENV['BETA_HOCKEY_API_TOKEN'], notify: '0', notes: ENV['RELEASE_NOTES'], notes_type: '0')

  end

  # You can define as many lanes as you want

  after_all do |lane|
    # This block is called, only if the executed lane was successful
    
    # slack(
    #   message: "Successfully deployed new App Update."
    # )
  end
```

### Environment

fastlane version (run `fastlane -v`): 1.97.2

Do you use bundler to execute fastlane (i.e. `bundle exec fastlane`)? No

Do you use a Ruby environment manager (e.g. `chruby`, `rbenv`, `rvm`)? rvm && travis

**Comment by a1cooke** [comment]
cheers fastlane-bot but it's an actual issue, I can't reproduce it locally but I'm not sure where to next in terms of how to fix this myself

**Comment by pahnev** [comment]
I just had the same issue with Jenkins running fastlane. Running same command locally worked fine.

**Comment by TKBurner** [comment]
Hi @pahnev can you share your full output?

@a1cooke Thanks for following up. Do you run into this issue if you unlock the keychain first?

**Comment by pahnev** [comment]
Here is my output
```
[12:50:21]: fastlane finished with errors
/Users/ci/.rvm/rubies/ruby-2.2.3/lib/ruby/site_ruby/2.2.0/rubygems/core_ext/kernel_require.rb:114:in `require': unable to find a version of 'krausefx-shenzhen' to activate (Gem::LoadError)
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/actions/hockey.rb:16:in `run'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/runner.rb:197:in `block (2 levels) in execute_action'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/actions/actions_helper.rb:35:in `execute_action'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/runner.rb:182:in `block in execute_action'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/runner.rb:181:in `chdir'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/runner.rb:181:in `execute_action'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/runner.rb:107:in `trigger_action_by_name'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/fast_file.rb:140:in `method_missing'
	from Fastfile:126:in `block (2 levels) in parsing_binding'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/lane.rb:33:in `call'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/lane.rb:33:in `call'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/runner.rb:50:in `block in execute'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/runner.rb:46:in `chdir'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/runner.rb:46:in `execute'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/lane_manager.rb:46:in `cruise_lane'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/command_line_handler.rb:30:in `handle'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/commands_generator.rb:51:in `block (2 levels) in run'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/commander-4.3.5/lib/commander/command.rb:178:in `call'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/commander-4.3.5/lib/commander/command.rb:153:in `run'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/commander-4.3.5/lib/commander/runner.rb:428:in `run_active_command'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane_core-0.48.1/lib/fastlane_core/ui/fastlane_runner.rb:26:in `run!'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/commander-4.3.5/lib/commander/delegates.rb:15:in `run!'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/commands_generator.rb:244:in `run'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/commands_generator.rb:20:in `start'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/lib/fastlane/cli_tools_distributor.rb:47:in `take_off'
	from /Users/ci/.rvm/gems/ruby-2.2.3@global/gems/fastlane-1.97.2/bin/fastlane:5:in `<top (required)>'
	from /Users/ci/.rvm/gems/ruby-2.2.3/bin/fastlane:23:in `load'
	from /Users/ci/.rvm/gems/ruby-2.2.3/bin/fastlane:23:in `<main>'
	from /Users/ci/.rvm/gems/ruby-2.2.3/bin/ruby_executable_hooks:15:in `eval'
	from /Users/ci/.rvm/gems/ruby-2.2.3/bin/ruby_executable_hooks:15:in `<main>'
Build step 'Execute shell' marked build as failure
Archiving artefacts
Finished: FAILURE
```

**Comment by a1cooke** [comment]
@TKBurner I worked with Travis support and unblocked myself, the solution was that json 2.1.0 is conflicting with fastlane.

Current solution was to uninstall the gem and go back to 1.8: `gem uninstall json --version 2.0.1`

**Comment by TKBurner** [comment]
@a1cooke Thanks so much for letting me know. I'll make sure to tell anyone that is similarly affected. Hope you have a great day! 👍
